gr-digital/grc/digital_constellation_receiver_cb.block.yml

```diff
@@ -59,4 +60,18 @@ templates:
     make: digital.constellation_receiver_cb(${constellation}, ${loop_bw}, ${fmin},
         ${fmax})
 
+cpp_templates:
+    includes: ['#include <gnuradio/digital/constellation_receiver_cb.h>']
+    declarations: 'digital::constellation_receiver_cb::sptr ${id};'
+    make: |-
+        this->${id} = digital::constellation_receiver_cb::make(
+            ${constellation},
+            ${loop_bw},
+            ${fmin},
+            ${fmax}));
+    link: ['gnuradio-digital']
+    translations:
```

Comment:
Actually, this block doesn't need `translations `at all. `showports` is not mentioned in the templates.